ftw/simplelayout/aliasblock/contents/aliasblock.py

```diff
@@ -0,0 +1,57 @@
+from ftw.referencewidget.sources import ReferenceObjSourceBinder
+from ftw.referencewidget.widget import ReferenceBrowserWidget
+from ftw.simplelayout import _
+from ftw.simplelayout.aliasblock.contents.interfaces import IAliasBlock
+from plone import api
+from plone.autoform.interfaces import IFormFieldProvider
+from plone.dexterity.content import Item
+from plone.directives import form
+from plone.directives.form import widget
+from z3c.relationfield.schema import RelationChoice
+from zope.interface import alsoProvides
+from zope.interface import implements
+
+
+def get_alias_path(widget):
+    return '/'.join(api.portal.get().getPhysicalPath())
```

Comment:
Why always start at the root?

Reply:
It doesn't say in the specifications and since it would be pointless to reference a block close to the page having the alias block I decided to just start at the root.
I guess that's something we need to configure for a customer if they have a certain preference and otherwise just go with something.

Reply:
@busykoala There is a quite hidden feature https://github.com/4teamwork/ftw.referencewidget/blob/d24ee07563f7f1657299b02a6376ca3630e9bb4c/ftw/referencewidget/browser/utils.py#L89 I would probably use "parent" 😊

Reply:
Nice, thanks!